### PR TITLE
rpc: Add default config options to settings.

### PR DESCRIPTION
**Pull request by JoeGruffins**
If user does not supply some default values over rpc when creating
    wallets, use default values. Add the eth test token, zec, and doge to
    the simnet client set up script.

This is mostly to assist testing on simnet.

### CODE REVIEW

File: client/core/core.go [hunk 1]
```diff
@@ -8403,9 +8403,9 @@ func parseCert(host string, certI interface{}, net dex.Network) ([]byte, error)
 	return nil, fmt.Errorf("not a valid certificate type %T", certI)
 }
 
-// walletDefinition gets the registered WalletDefinition for the asset and
+// WalletDefinition gets the registered WalletDefinition for the asset and
 // wallet type.
-func walletDefinition(assetID uint32, walletType string) (*asset.WalletDefinition, error) {
+func WalletDefinition(assetID uint32, walletType string) (*asset.WalletDefinition, error) {
```
Comment (chappjc) [inline client/core/core.go:8408]: Kinda funky that this is a `core` functions.  It only uses the `client/asset` package internally, and it returns a `client/asset` type.

**Comment by JoeGruffins** [comment]
`WalletDefinition` moved to client/asset https://github.com/decred/dcrdex/compare/611f2faaad4d2d1902a97224b722d510a4567f23..78da72663ca4d753712a380555465ec54b947217

File: client/rpcserver/handlers.go [hunk 2]
```diff
@@ -192,6 +193,22 @@ func handleNewWallet(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
...
+		errMsg := fmt.Sprintf("error creating %s wallet: unable to get wallet definition: %v",
+			dex.BipIDSymbol(form.assetID), err)
+		resErr := msgjson.NewError(msgjson.RPCWalletDefinitionError, errMsg)
+		return createResponse(newWalletRoute, nil, resErr)
+	}
+
+	// Apply default config options if they exist.
+	for _, opt := range walletDef.ConfigOpts {
+		if _, has := form.config[opt.Key]; !has && opt.DefaultValue != nil {
+			form.config[opt.Key] = fmt.Sprintf("%v", opt.DefaultValue)
```
Comment (chappjc) [inline client/rpcserver/handlers.go:208]: `fmt.Sprintf("%v", ...)` should do the trick for all the types we give in `DefaultValue`, but in our other uses of this field (`WalletConfigForm.map`) we consider the other flags like `isdate`and `repeatable`.  I don't feel like we need to replicate all that at this time though.

**Review by chappjc — APPROVED** [review]
LGTM, with the one noted caveat regarding the correct way to go from a configopt item to a string in the `map[string]string` given to the wallet form.

File: client/rpcserver/handlers.go (hunk 2 above)
Comment (chappjc) [reply client/rpcserver/handlers.go:208]: To clarify, nothing to change unless another reviewer sees an issue related to this.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**